bodyshopbidsdotcom/git-scripts
# Lowercase Queries to ECR Repos

ECR repository names need to be lowercase. This was fixed in our scripts for resource creation, but it wasn't added to the script that queries them.

### Testing

1. Checkout the branch `w270870137-add-dockerfile-to-project` of the [Mammon](https://github.com/bodyshopbidsdotcom/Mammon) project (note the casing in the repository name).
1. Checkout the `release` branch of `git-scripts`.
1. Run `git_run_eb stage rails c` and confirm that a `InvalidParameterException` occurs.
1. Check out this branch, run the command, and confirm that a rails console process starts.
